R/process_extract_cmh.R

```diff
@@ -21,56 +21,62 @@ process_extract_cmh <- function(data,
   # Check that the supplied year is in the correct format
   year <- check_year_format(year)
 
-  # Data Cleaning  ---------------------------------------
+  # If data is available in the FY then run processing.
+  # If no data has passed through, return NULL.
+  if (is.null(data)) {
+    outfile <- NULL
+  } else {
```

Comment:
I think this would make the code read a bit easier?

```suggestion
  # If no data has passed through, return NULL.
  if (is.null(data)) {
    return(NULL)
  } 
```
(You would also need to delete the last `}`)